tests/test_admin.py

```diff
@@ -216,3 +218,106 @@ def test_except_cred_list(client):
     assert result.output == (
         f"Credentials listing failed. \n" f"'NoneType' object has no attribute 'user'\n"
     )
+
+
+def fake_clients():
+    user = [
+        {
+            "userId": "jerrygarcia",
+            "fullName": "Jerry Garcia",
+            "emailAddr": "garcia@bgn.net",
+            "address1": "456 Shakedown St.",
+            "city": "Portsmouth",
+            "active": True,
+        },
+        {
+            "userId": "johnthompson",
+            "fullName": "John Thompson",
+            "emailAddr": "",
+            "address1": "",
+            "city": "",
+            "active": True,
+        },
+    ]
+    client = mock.Mock()
+    client.user.list.return_value = user
+    client.user.side_effect = client.user.list().pop(1)
+    return client
+
+
+def test_remove_user(monkeypatch):
+    monkeypatch.setattr(admin_client, "get_admin_client", fake_clients)
+    monkeypatch.setattr(obs.libs.utils, "check", lambda response: None)
+
+    runner = CliRunner()
+    result = runner.invoke(cli, ["admin", "user", "rm"])
```

Comment:
This will always be True, even though this is the wrong command.

```
$ obs admin user rm
User removal failed.
Bad Request [400]
URL: http://103.77.104.212:19443/user?userId=None&groupId=None
```